apitable.go

```diff
@@ -162,3 +190,14 @@ func (c *WTableCell) Shade(val, color, fill string) *WTableCell {
 	}
 	return c
 }
+
+//Replaces any index that is blank with "#000000"
+func CheckBorderColors (borderColors [6]string) [6]string {
```

Comment:
This can be private.

Reply:
Removed all after changing to struct

Reply:
So you can remove this change. But the default value is still `""` so you can add a private method to the struct `TableBorderColors` like
```go
func (tbc *TableBorderColors) applydefault() {
    // look each field and fill "#000000" to "" value
}
```